Integration test failures

**Describe the bug**

Found by @rwood-moz:

> I'm seeing some Appointment integration test failures that are new (they used to all pass for me when running locally).
```
======================================================================================= short test summary info ========================================================================================
FAILED test/integration/test_schedule.py::TestRequestScheduleAvailability::test_fail_not_utc - TypeError: cannot pickle 'FrameLocalsProxy' object
FAILED test/integration/test_schedule.py::TestRequestScheduleAvailability::test_fail_slot_duration_mismatch - TypeError: cannot pickle 'FrameLocalsProxy' object
FAILED test/integration/test_schedule.py::TestRequestScheduleAvailability::test_fail_on_invalid_weekday - TypeError: cannot pickle 'FrameLocalsProxy' object
FAILED test/integration/test_schedule.py::TestRequestScheduleAvailability::test_fail_on_before_time_range - TypeError: cannot pickle 'FrameLocalsProxy' object
FAILED test/integration/test_schedule.py::TestRequestScheduleAvailability::test_fail_on_after_time_range - TypeError: cannot pickle 'FrameLocalsProxy' object
============================================================================== 5 failed, 19 passed, 209 warnings in 2.81s ==============================================================================
```

**Additional context**
Note that in CI the tests are passing fine, so maybe it's something with my local env somehow.

--- a/backend/requirements.txt
+++ b/backend/requirements.txt
@@ -20,7 +20,7 @@ python-dotenv==1.0.1
 python-multipart==0.0.18
 PyJWT==2.6.0
 pydantic[email]==2.9.2
-sentry-sdk==2.10.0
+sentry-sdk==2.20.0
 starlette-context==0.3.6
 sqlalchemy-utils==0.41.2
 sqlalchemy==2.0.36

